lms/config.py

```diff
@@ -39,7 +39,7 @@ class _Setting:
     """The properties of a setting and how to read it."""
 
     name: str
-    read_from: str | None = None
+    read_from: str = None  # type: ignore
```

Comment:
We use this pattern, allowing None in a dataclass field to provide a default in `__post_init__`


Ignore the type here to avoid multiple issues about it being nullable all over the place.